strategies/changed/src/main/java/org/arquillian/smart/testing/vcs/git/GitChangesDetector.java

```diff
@@ -98,6 +98,10 @@ boolean matchPatterns(String path) {
         return false;
     }
 
+    boolean matchMainClassesPattern(String path) {
+        return matchPattern(path, "**/src/main/java/**/*.java");
```

Comment:
I think, we should improve this in future - https://github.com/arquillian/smart-testing/blob/master/strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/AffectedChangesDetectorFactory.java#L31